src/reducers/versions.tsx

```diff
@@ -195,13 +195,16 @@ type VersionAddon = {
 export type Version = {
   addon: VersionAddon;
   entries: VersionEntry[];
-  expandedPaths: string[];
   id: number;
   reviewed: string;
-  selectedPath: string;
   validationURL: string;
-  visibleSelectedPath: string | null;
   version: string;
+  // TODO: remove `expandedPaths`, `selectedPath` and `visibleSelectedPath`
+  // from `version` object once no code depends on these
+  // https://github.com/mozilla/addons-code-manager/pull/1204
```

Comment:
I made https://github.com/mozilla/addons-code-manager/issues/1218 so we don't lose track of it:

```suggestion
  // See https://github.com/mozilla/addons-code-manager/issues/1218
```